optimum/intel/neural_compressor/quantization.py

```diff
@@ -580,7 +578,7 @@ def from_pretrained(cls, model_name_or_path: str, q_model_name: Optional[str] =
         except Exception:
             logger.info("Couldn't verify torch version.")
 
-        if getattr(config, "backend", None) == "ipex":
+        if getattr(config, "torchscript", False):
```

Comment:
Will a model obtained with a previous optimum version would still be compatible ? otherwise we can have :
```suggestion
        if getattr(config, "backend", None) == "ipex" or getattr(config, "torchscript", False):
```

Reply:
done